### ISSUE TITLE
directions to train OMCNN correctly

### ISSUE THREAD

**Issue by prashnani** [remega/OMCNN_2CLSTM#12]
Congrats for the excellent work and thank you for the well-documented public release of dataset! 

Could you please provide the training code? It would be helpful to understand and reproduce some of the useful steps mentioned in your paper, such as CB dropout, etc.

**Comment by lhj849327832** [comment]
ME TOO
